packages/website/src/js/fetchGitHubRepoData.js

```diff
@@ -0,0 +1,61 @@
+const githubStarsText = body.querySelector("#github-stars")
+const githubDependentsText = body.querySelector("#github-dependents")
+
+const githubPackageDependentsURL =
+  "https://github.com/TrueFiEng/useDApp/network/dependents?package_id="
+const githubPackageID = "UGFja2FnZS0xODg5OTgyMTY5"
+const proxyURL = "https://api.allorigins.win/raw?url="
+
+const showFetchedStars = async () => {
+  githubStarsText.textContent = "1.2k+"
+
+  await fetch("https://api.github.com/repos/TrueFiEng/useDApp")
+    .then((response) => response.json())
+    .then(
+      (data) =>
+      (githubStarsText.textContent =
+        Intl.NumberFormat("en-US", { notation: "compact" })
+          .format(data.stargazers_count)
+          .toLowerCase() + "+")
```

Comment:
```suggestion
          .toLowerCase())
```

We already have the word `Over` so the plus duplicates it.